test/functionalTest/cases/4286_Update_format_for_notification_messages/Update_format_for_notification_messages.test

```diff
@@ -0,0 +1,203 @@
+# Copyright 2023 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Update format for notification messages
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB 0
+accumulatorStart --pretty-print
+
+--SHELL--
+
+#
+# 01. Create NGSIv2 subscription for E1/A1 (A1, A4), with attrsFormat set to simplifiedNormalized
+# 02. Create NGSIv2 subscription for E1/A1 (A1, A5), with attrsFormat set to simplifiedkeyValues
+# 03. Create E1/A1
+# 04. Dump accumulator to see notifications
+#
+
+
+echo "01. Create NGSIv2 subscription for E1/A1 with attrsFormat set to simplifiedNormalized"
+echo "====================================================================================="
+payload='
+{
+  "subject": {
+    "entities": [
+      {
+        "id": "E1"
+      }
+    ],
+    "condition": {
+      "attrs": [ "A1" ]
+    }
+  },
+  "notification": {
+    "httpCustom": {
+      "url": "http://localhost:'$LISTENER_PORT'/notify"
+    },
+    "attrs": [ "A1", "sub1_with_simplifiedNormalized" ],
```

Comment:
```suggestion
```

You are not using `sub1_with_simplifiedNormalized` so I understand it is not needed.

Reply:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/4298/commits/1de26680286740a30ffeee899efe749412072f20